vmm_tests/vmm_tests/tests/tests/x86_64/openhcl_linux_direct.rs

```diff
@@ -396,6 +396,90 @@ async fn openhcl_linux_storvsp_dvd(config: PetriVmConfigOpenVmm) -> Result<(), a
     Ok(())
 }
 
+/// Test an OpenHCL Linux direct VM with a SCSI DVD assigned to VTL2, using NVMe
+/// backing, and vmbus relay. This should expose a DVD to VTL0 via vmbus.
+#[openvmm_test(openhcl_linux_direct_x64)]
+async fn openhcl_linux_storvsp_dvd_nvme(config: PetriVmConfigOpenVmm) -> Result<(), anyhow::Error> {
+    const NVME_INSTANCE: Guid = guid::guid!("dce4ebad-182f-46c0-8d30-8446c1c62ab3");
+    let vtl2_nsid = 1;
+    let nvme_disk_sectors: u64 = 0x10000;
+    let sector_size = 512;
```

Comment:
Please make `sector_size` 4k (and reduce `nvme_disk_sectors appropriately)

Reply:
Done in my local working copy